source/EvoPlanet/evoplanet.client/src/app/components/simulator/simulator.component.css

```diff
@@ -0,0 +1,27 @@
+.simulator-container {
+  width: 100%;
+  height: 91vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+#simulator-canvas {
+  border: 1px solid black;
+
+}
+.simulatortohome {
```

Comment:
A question, not a request: If I remember correctly, we are already using either bootstrap or Angular Material in the project. You could have used button formatting on *simulatortohome*, and achieved a similar result. It's completely fine as it is, though it looks a bit like Bootstrap's `btn-secondary` formatting option, which you could have used.

You don't need to change anything now, I just want to make sure you know using existing styles is an option.